src/Chunk.h

```diff
@@ -439,7 +439,12 @@ class cChunk :
 	void SetAlwaysTicked(bool a_AlwaysTicked);
 
 	// Makes a copy of the list
-	cClientHandleList GetAllClients(void) const {return m_LoadedByClient; }
+	cClientHandleList GetAllClients(void) const
+	{
+		cClientHandleList copy;
+		std::copy(m_LoadedByClient.begin(), m_LoadedByClient.end(), std::back_inserter(copy));
+		return copy;
```

Comment:
That would be a better way of doing it. Shall I add that to the improvements PR I'll write tomorrow, along with and better handling for duplicate clienthandles?